cfg/cfg.go

```diff
@@ -24,6 +22,54 @@ type Config struct {
 	Validation_level_legacy validate.LevelLegacy
 	Validation_level_m20    validate.LevelM20
 	Validate_order          bool
+	BlackList               []string
+	Aggregation             []*AggregationConfig
+	Route                   []*RouteConfig
+	Rewriter                []*RewriterConfig
+}
+
+type AggregationConfig struct {
+	Type     string
```

Comment:
Sure, not sure where I got Type from as I did generally try to keep things consistent.